### PR TITLE
Ability to configure the server DB connection pool from the custom resource (1.11.x)

### PR DESCRIPTION
**Pull request by claudio4j**
https://issues.redhat.com/browse/ENTESB-14654

### CODE REVIEW

File: install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml [hunk 1]
```diff
@@ -2226,6 +2226,36 @@ spec:
                     type: object
                   server:
                     properties:
+                      connectionPool:
+                        description: Connectin Pool parameters used in syndesis-server
```
Comment (KurtStam) [inline install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml:2230]: Connection pool - and we probably want to remove the link reference to hikariCP

File: install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml [hunk 2]
```diff
@@ -2226,6 +2226,36 @@ spec:
...
+                          maxLifetime:
+                            description: maximum lifetime of a connection in the pool
+                            type: integer
+                          maximumPoolSize:
+                            description: maximum size that the pool is allowed to
+                              reach, including both idle and in-use connections
+                            type: integer
+                          minimumIdle:
+                            description: minimum number of idle connections that HikariCP
+                              tries to maintain in the pool
```
Comment (KurtStam) [inline install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml:2256]: minimum number of idle connections maintained in the pool

**Review by KurtStam — CHANGES_REQUESTED** [review]

File: install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml (hunk 2 above)
Comment (claudio4j) [reply install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml]: Fixed. Thanks.

File: install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml (hunk 1 above)
Comment (claudio4j) [reply install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml]: Fixed. Thanks.

**Pull request merged**